roles/nfd_operator_check_resource_statuses/tasks/clusterrole_init.yml

```diff
@@ -0,0 +1,44 @@
+---
+# If no ClusterRoles were found at all, then something is wrong.
+- name: Search to see if the NFD Operator ClusterRoles exist
+  fail: msg="No ClusterRoles were found"
+  when: all_nfd_operator_manifests.results | selectattr(search_key,'contains',search_val) | list | count == 0
+  vars:
+    search_key: 'stdout'
+    search_val: '{{ is_clusterrole }}'
+
+# Get Role names
+- name: Get the NFD Operator's ClusterRole names from the manifests
+  shell: |
+    set -o pipefail;
+    cat '{{ clusterrole_manifest.manifest.path }}' | shyaml get-value metadata.name
```

Comment:
I'm not sure what is `shyaml`, but it's not yet part of the image for sure
we usually use `cat $file.yml | yq | jq ....` to convert yaml to json (`yq`), and then select from json (`jq`)